tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -656,7 +656,7 @@ public bool LogsInjectionEnabled
             PublicApiUsage.TracerSettings_GlobalTags_Get,
             PublicApiUsage.TracerSettings_GlobalTags_Set)]
         [ConfigKey(ConfigurationKeys.GlobalTags)]
-        internal IDictionary<string, string> GlobalTagsInternal { get; private set; }
+        internal IDictionary<string, string?> GlobalTagsInternal { get; private set; }
```

Comment:
These should not be changed IMO. This essentially changes the public API to something we _don't_ support.